tests/monotouch-test/HealthKit/HKHealthStoreTest.cs

```diff
@@ -0,0 +1,42 @@
+//
+// Unit tests for HKHealthStore
+//
+// Authors:
+//	TJ Lambert  <TJ.Lambert@microsoft.com>
+//
+// Copyright 2022 Xamarin Inc. All rights reserved.
+//
+
+#if HAS_HEALTHKIT
+
+using System;
+
+using Foundation;
+using HealthKit;
+using UIKit;
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.HealthKit {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class HKHealthStoreTest {
+
+		[Test]
+		public void GetBiologicalSexNullReturnTest ()
```

Comment:
These tests are here because the docs said the returns were not nullable, but the headers did say they were. The tests show that they are in fact nullable